regression/cbmc/CMakeLists.txt

```diff
@@ -1,41 +1,40 @@
 if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
   set(gcc_only -X gcc-only)
+  set(gcc_only_string ";-X;gcc-only")
```

Comment:
I have two questions about this PR:

1. Why is this being added? I can see that this is moved to be back into the quotes part in line 16, but I don't understand why this being done.
2. I hope I'm not being overly nitpicky, but this string reads a little bit weird to me. May I suggest that this is turned into something like `-X;gcc-only;` so that it looks more naturally delimited here and in line 16? Unless there's a specific reason for it to be like this.

Reply:
Re 1) The macro `add_test_pl_profile` takes exactly four arguments when previously six were passed (an additional "-X" and "gcc-only"). So `gcc_only` was simply ignored.
Re 2) Fair point, I've fixed this.